tests/forms_tests/tests/test_formsets.py

```diff
@@ -1553,6 +1553,28 @@ def test_custom_renderer(self):
         self.assertEqual(formset.non_form_errors().renderer, renderer)
         self.assertEqual(formset.empty_form.renderer, renderer)
 
+    def test_form_default_renderer(self):
+        """
+        In the absense of a renderer passed to formset_factory, the default_renderer
+        attribute of the Form class should be respected.
+        """
+        from django.forms.renderers import Jinja2
+
+        class ChoiceWithDefaultRenderer(Choice):
+            default_renderer = Jinja2()
```

Comment:
Can we use `CustomRenderer` instead? For example:
```python
class CustomRenderer(DjangoTemplates):
    pass

class ChoiceWithDefaultRenderer(Choice):
    default_renderer = CustomRenderer
```

Reply:
I believe we could. I'm curious as to why that would be preferable?

Reply:
Because these tests are run also on `Jinja2`, so `test_form_default_renderer` crashes on jinja2 just because it's a different instance, but the class is the same.